dot/network/notifications.go

```diff
@@ -201,6 +204,7 @@ func (s *Service) createNotificationsMessageHandler(info *notificationsProtocol,
 				info.inboundHandshakeData.Store(peer, hsData)
 
 				// once validated, send back a handshake
+				// TODO: getHandshake creates a new handshake struct. Rename?
```

Comment:
`getHandshake` is just some function that satisfies the handshake getter function signature, it doesn't necessarily make a new struct, and could be different for each notifications protocol